docs/docs/databases/ms-sql-server/getting-started/cloud.mdx

```diff
@@ -59,17 +59,17 @@ Once you add the database, you'll see your database pop up on the sidebar.
 If you have existing tables, head to the database page by clicking on the database name on the sidebar. You should see a
 list of tables.
 
-<Thumbnail src='/img/databases/ms-sql-server/3-manage-mydb.png' alt='Manage my-db' width='1000px' />
+<Thumbnail src='/img/databases/ms-sql-server/manage-db_step-3_console_2.10.1.png' alt='Manage my-db' width='1000px' />
 
 Track tables selectively or all of them so that Hasura can introspect the tables and create the corresponding GraphQL
 schema.
 
-<Thumbnail src='/img/databases/ms-sql-server/4-track-tables.png' alt='Track tables' width='1000px' />
+<Thumbnail src='/img/databases/ms-sql-server/track-tables_step-3_console_2.10.1.png' alt='Track tables' width='1000px' />
 
 If you have foreign keys, you'll also see suggested relationships. Again, you can choose to track them selectively or
 all at once.
 
-<Thumbnail src='/img/databases/ms-sql-server/5-track-rels.png' alt='Track relationships' width='1000px' />
+<Thumbnail src='/img/databases/ms-sql-server/track-foreign-key-rel_step-3_console_2.10.1.png' alt='Track relationships' width='1000px' />
```

Comment:
Also too wide. Set to actual size of the UI. Any chance we could rename "People" to "Users" it looks more professional. 

<img width="1728" alt="Screenshot 2022-10-18 at 14 28 41" src="https://user-images.githubusercontent.com/94021366/196443591-7232e00c-21f0-4a1a-ba59-99da8929c17a.png">